src/Persistence/Array_.php

```diff
@@ -31,15 +31,9 @@ public function __construct(array $data = [])
      */
     protected $lastInsertIds = [];
 
-    /**
-     * @deprecated TODO temporary for these:
-     *             - https://github.com/atk4/data/blob/90ab68ac063b8fc2c72dcd66115f1bd3f70a3a92/src/Reference/ContainsOne.php#L119
-     *             - https://github.com/atk4/data/blob/90ab68ac063b8fc2c72dcd66115f1bd3f70a3a92/src/Reference/ContainsMany.php#L66
-     *             remove once fixed/no longer needed
-     */
-    public function getRawDataByTable(string $table): array
+    public function getRawDataIterator($table): \Iterator
```

Comment:
👍 for removing `getRawDataByTable()`

Reply:
Question here is if we introduce an Interface for the persistence with 3 methods
`getRawDataIterator`
`setRawData`
`unsetRawData`

Now used in both `Array_` and `Csv`

Reply:
we should never work on raw data, as the structure can be completely different, even if "Array" persistence and we was almost there...

Reply:
Only Query objects work on raw data. For sql we use the link to SQL server, for other persistences the Persistence object provides access to raw data.
If you figure out a way to limit the access nicely then we can implement.